dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -307,6 +319,10 @@ def mean(self):
             return 0
         return float(self.sum) / self.match_count
 
+    @property
+    def mode(self):
+        return self._estimate_mode_from_histogram()
```

Comment:
this is abstracting it, but do you think it is necessary?

Also, add a function description.

Reply:
I think there could be potential future features that may need access to the mode, in which case it could be easier to call and use with this.